src/styles/card/card.scss

```
@@ -0,0 +1,78 @@
+@import "../utils/mixins.scss";
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  /*#region img*/
+
+  &__image {
+    text-align: center;
+    margin-bottom: 37px;
+  }
+  &__image > img {
+    width: 160px;
+    height: 134px;
+  }
+  /*#endregion*/
+  /*#region text*/
+  &__text {
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code,
```

```suggestion
  &__text {
    font-weight: 500;
    line-height: 18px;
    margin-bottom: 4px;
  }
  
  &__code,
```
add blanks between selectors, it will increase readability